x/wasm/client/cli/query.go

```diff
@@ -19,8 +19,8 @@ import (
 	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
-	"github.com/cosmwasm/wasmd/x/wasm/internal/keeper"
```

Comment:
They decided to change the "standard module structure" again?

Reply:
sadly, yes. There's no differentiation of internal and external anymore